teoriasegnali/fourier.tex

```diff
@@ -45,44 +44,46 @@ \section{Condizioni di esistenza Dirichlet}\index{serie!di Fourier!condizioni Di
 
 \section{Serie di Fourier di sinusoidi}
 Lo sviluppo in serie di Fourier può essere espresso come serie somma di seni e coseni o sinusoidi
-\[\begin{split} s(t)&=\sum_{k=0}^{+\infty}\left[ a_k \sen{2\pi\frac{k}{T}t} + b_k \cos{2\pi\frac{k}{T}t}\right] \\
-&=\sum_{k=0}^{+\infty}A_k\left[\sen{2\pi\frac{k}{T}t+\phi_k}\right] =\sum_{k=0}^{+\infty}A_k\left[\sen{2\pi\frac{k}{T}t}\Cos\phi_k +\cos{2\pi\frac{k}{T}t}\Sen\phi_k \right]
+\[\begin{split} s(t) &= \sum_{k=0}^{+\infty}\bigl( a_k \sen{2\pi kf_0 t} + b_k \cos{2\pi kf_0 t}\bigr) = \\
```

Comment:
Sì, è vero. Il fine dell'uso di big[l/r] è per rendere le parentesi esterne più in vista rispetto quelle interne dato che l'espressione ha la stessa altezza, ma se non sei d'accordo posso comunque ripristinare le modifiche.